apps/files/src/views/FilesList.vue

```diff
@@ -294,6 +295,16 @@ export default defineComponent({
 				return []
 			}
 
+			let filteredDirContent = [...this.dirContents]
+			// Filter based on the provided filterText
+			if (this.filterText) {
+				filteredDirContent = filteredDirContent.filter(content => {
+					// Adjust this condition based on your specific filtering criteria
+					return content.attributes[this.sortingMode].toLowerCase().includes(this.filterText.toLowerCase())
+				})
+				console.debug('Files view filtered', filteredDirContent)
+			}
```

Comment:
Why filter by attribute? :thinking: 

```suggestion
			let filteredDirContent = [...this.dirContents]
			// Filter based on the provided filterText
			if (this.filterText) {
				filteredDirContent = filteredDirContent.filter(node => {
					// Adjust this condition based on your specific filtering criteria
					return node.basename.toLowerCase().includes(this.filterText.toLowerCase())
				})
				console.debug('Files view filtered', filteredDirContent)
			}
```

Reply:
I see that is it configurable what column is used for sorting or searching. `this.sortingMode` defaults to `'basename'` and `content.attributes[this.sortingMode]` evaluates to `content.attributes.basename`

But `node` sounds like better naming.

Reply:
This means, if the user changes the sorting mode, their search would use a different column.

Reply:
No, if I read the code properly, it means you would filter the current sorting column ? :thinking: 
Search should always filter by basename ?